src/main/java/com/gizasystems/automationexercise/pages/RecommendedSection.java

```diff
@@ -0,0 +1,51 @@
+package com.gizasystems.automationexercise.pages;
+
+import com.shaft.driver.SHAFT;
+import io.qameta.allure.Step;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+public class RecommendedSection {
+    // Variables
+    private SHAFT.GUI.WebDriver driver;
+
+    // Locators
+    private final By productAddedToCartMessage = By.cssSelector("div.modal-content > div > h4");
+    private final By recommendedItems_div = By.cssSelector("div.recommended_items");
+
+    // Constructor
+    public RecommendedSection(SHAFT.GUI.WebDriver driver){
+        this.driver = driver;
+    }
+    @Step("Navigate Recommended Section Tab")
+    public RecommendedSection openRecommendedSection(){
+        JavascriptExecutor javascriptExecutor = (JavascriptExecutor) driver.getDriver();
+        WebElement webElement = driver.getDriver().findElement(recommendedItems_div);
+        javascriptExecutor.executeScript("arguments[0].scrollIntoView({behavior: 'smooth', block: 'center', inline: 'center'})",webElement);
+        Actions actions = new Actions(driver.getDriver());
+        actions.pause(2000).moveToElement(webElement).perform();
+        return this;
+    }
+    //////////////////// Actions \\\\\\\\\\\\\\\\\\\\
+    @Step("Add Recommended Product To Cart")
+    public RecommendedSection addToCart(String productName){
+        String productNameValue = "//div[@class='recommended_items']//child::p[text()='" + productName + "']//parent::div//a";
+        driver.element().click(By.xpath(productNameValue));
```

Comment:
We should not add the locator in the method itself. Please create a method for the locator that returns By object, then use it here.